File: delivery_API/classes/Livraison.php
<?php

class Livraison
{
    private $client_id;
    private $date_livraison;
    private $livreur_id;
    private $adresse_depart_id;
    private $adresse_arrivee_id;
    private $statut;

    public function __construct($client_id, $date_livraison, $livreur_id, $adresse_depart_id, $adresse_arrivee_id, $statut = 'cree')
    {
        $this->client_id = $client_id;
        $this->date_livraison = $date_livraison;
        $this->livreur_id = $livreur_id;
        $this->adresse_depart_id = $adresse_depart_id;
        $this->adresse_arrivee_id = $adresse_arrivee_id;
        $this->statut = $statut;
    }

    public function getClientId()
    {
        return $this->client_id;
    }
    public function setClientId($client_id)
    {
        $this->client_id = $client_id;
    }

    public function getDateLivraison()
    {
        return $this->date_livraison;
    }
    public function setDateLivraison($date_livraison)
    {
        $this->date_livraison = $date_livraison;
    }

    public function getLivreurId()
    {
        return $this->livreur_id;
    }
    public function setLivreurId($livreur_id)
    {
        $this->livreur_id = $livreur_id;
    }

    public function getAdresseDepartId()
    {
        return $this->adresse_depart_id;
    }
    public function setAdresseDepartId($adresse_depart_id)
    {
        $this->adresse_depart_id = $adresse_depart_id;
    }

    public function getAdresseArriveeId()
    {
        return $this->adresse_arrivee_id;
    }
    public function setAdresseArriveeId($adresse_arrivee_id)
    {
        $this->adresse_arrivee_id = $adresse_arrivee_id;
    }

    public function getStatut()
    {
        return $this->statut;
    }
    public function setStatut($statut)
    {
        $this->statut = $statut;
    }
}

File: Delivery_MOBILE/app/src/main/java/com/example/campusdelivery/classes/Livraison.java
package com.example.campusdelivery.classes;

public class Livraison {

    private int client_id;
    private int livreur_id;
    private String adresse_depart;
    private String adresse_arrivee;
    private String statut;
    private String date_livraison;

    public Livraison(int client_id, int livreur_id,
                     String adresse_depart, String adresse_arrivee,
                     String statut, String date_livraison) {

        this.client_id = client_id;
        this.livreur_id = livreur_id;
        this.adresse_depart = adresse_depart;
        this.adresse_arrivee = adresse_arrivee;
        this.statut = statut;
        this.date_livraison = date_livraison;
    }

    public int getClient_id() { return client_id; }
    public int getLivreur_id() { return livreur_id; }
    public String getAdresse_depart() { return adresse_depart; }
    public String getAdresse_arrivee() { return adresse_arrivee; }
    public String getStatut() { return statut; }
    public String getDate_livraison() { return date_livraison; }
}